charts/rstudio/source/Dockerfile

```diff
@@ -46,20 +46,20 @@ RUN install2.r -e shiny rmarkdown shinythemes shinydashboard && \
     mkdir -p /var/log/shiny-server && \
     chown rstudio:rstudio /var/log/shiny-server && \
     chmod go+w -R /var/log/shiny-server /usr/local/lib/R /srv /var/lib/shiny-server && \
-    chmod ugo+rwx -R /usr/lib/rstudio-server/www
+    chmod ugo+rwx -R /usr/lib/rstudio-server/www && \
+    echo 'r-libs-user=~/R/library' >>/etc/rstudio/rsession.conf && \
+    echo "R_LIBS=\${R_LIBS-'/home/rstudio-server/R/library'}" >/usr/local/lib/R/etc/Renviron.site
 
 COPY start.sh /usr/local/bin/start.sh
 
 RUN rstudio-server verify-installation
 
 RUN chmod -R go+rwX /home /home/rstudio /tmp/downloaded_packages /var/run/rstudio-server /var/lib/rstudio-server /var/log/rstudio && \
-    rm /var/lib/rstudio-server/rstudio-os.sqlite /var/run/rstudio-server/rstudio-rsession/rstudio-server-d.pid && \
-    echo 'r-libs-user=~/R/library' >>/etc/rstudio/rsession.conf && \
-    echo "R_LIBS=\${R_LIBS-'/home/rstudio-server/R/library'}" >/usr/local/lib/R/etc/Renviron.site
+    rm /var/lib/rstudio-server/rstudio-os.sqlite /var/run/rstudio-server/rstudio-rsession/rstudio-server-d.pid
 
 USER $APP_UID:$APP_GID
 WORKDIR $HOME
-EXPOSE 8787 3838
```

Comment:
Why was the port removed? It is still in the service

Reply:
Service is used for nginx configmap. No need to be exposed to avoid creating a route directly to the Rstudio app